### ISSUE TITLE
Color by base modification resets Group by tag

### ISSUE THREAD

**Issue by northwestwitch** [igvteam/igv.js#1877]
We have noticed this strange behaviour.

To reproduce:

1. Use group by tag, for instance HP:

<img width="201" alt="image" src="https://github.com/user-attachments/assets/307b08b4-9996-4ae5-8f69-d3dc32acbb5f">

<img width="509" alt="image" src="https://github.com/user-attachments/assets/4e89b1b6-5469-4496-b04a-84cd4f9e0df4">

<hr>

2. If you then click on Color by base modification:
 
<img width="208" alt="image" src="https://github.com/user-attachments/assets/0e8ce741-9905-4161-b5ad-aec45bfe7156">

You'll lose your grouping and gets grouped by read strand instead

<img width="181" alt="image" src="https://github.com/user-attachments/assets/3eb4c53a-cc49-4afd-8ce4-865d1acfb2d3">

Thanks for your help!

**Comment by jrobinso** [comment]
Yes that's actually as designed, but in retrospect not a great idea.   We will fix it.

**Comment by northwestwitch** [comment]
Thanks! 🙏🏻
